src/tests/end-to-end/common/browser-factory.ts

```diff
@@ -58,6 +68,12 @@ async function launchNewBrowser(): Promise<Puppeteer.Browser> {
     // end shows up as a generic timeout error with no meaningful logging".
     await verifyExtensionIsBuilt(extensionPath);
 
+    const browserLogDir = browserLogPath(browserInstanceId);
+    const userDataDir = path.join(browserLogDir, 'userDataDir');
+
+    // If this isn't present when chrome launches, it'll silently fail to log anything
+    await util.promisify(fs.mkdir)(userDataDir, { recursive: true });
```

Comment:
Does this comment mean that Chrome will silently fail to log anything if the directory doesn't exist? It was a little confusing when I first read it, but maybe adding an explanation for what happens when it is present (i.e. "Create directory so Chrome will log errors").

Reply:
Clarified the comment